### PR TITLE
Remove hardcoded modules

### PR DESCRIPTION
**Pull request by yurii-klymenko**

### CODE REVIEW

File: stix_shifter/scripts/stix_shifter.py [hunk 1]
```diff
@@ -147,7 +149,24 @@ def main():
 
     args = parent_parser.parse_args()
 
-    if args.command is None:
+    help_and_die = args.command is None
```
Comment (mdazam1942) [inline stix_shifter/scripts/stix_shifter.py]: `help_and_exit` ?

Comment (yurii-klymenko) [reply stix_shifter/scripts/stix_shifter.py]: has been renamed

File: stix_shifter_modules/carbonblack/stix_translation/data_mapping.py [hunk 2]
```diff
@@ -10,8 +10,8 @@ def __init__(self, options):
         assert "process" in self.map_data and "binary" in self.map_data
 
     def fetch_mapping(self):
-        process_mapping = self._fetch_mapping_file("process_from_stix_map.json")
-        binary_mapping = self._fetch_mapping_file("binary_from_stix_map.json")
+        process_mapping = self._fetch_mapping_file("process_from_stix_map_part.json")
```
Comment (mdazam1942) [inline stix_shifter_modules/carbonblack/stix_translation/data_mapping.py]: wondering why it is necessary to change the file name? basically you added `_part`

File: stix_shifter/scripts/stix_shifter.py (hunk 1 above)
Comment (mdazam1942) [reply stix_shifter/scripts/stix_shifter.py]: thanks :)

File: stix_shifter/scripts/stix_shifter.py [hunk 3]
```diff
@@ -6,12 +6,13 @@
...
 import time
 from stix_shifter_utils.utils.proxy_host import ProxyHost
+from stix_shifter_utils.utils.module_discovery import module_list, process_dialects
 
 TRANSLATE = 'translate'
 TRANSMIT = 'transmit'
 EXECUTE = 'execute'
 HOST = 'host'
-
+CONNECTOR_MODULES = module_list()
```
Comment (mdazam1942) [inline stix_shifter/scripts/stix_shifter.py]: does this need to be global? looks like only usage of this variable is to decide `help_and_exit` flag

File: stix_shifter_modules/carbonblack/stix_translation/data_mapping.py (hunk 2 above)
Comment (yurii-klymenko) [reply stix_shifter_modules/carbonblack/stix_translation/data_mapping.py]: carbonblack is the only connector that combines two mapping into the default no name dialect. Dialect discovery logic is based on the filenames, so for the carbonblack it has to find no mapping files for the dialects so only the default dialect is represented for it.

**Pull request merged**